database/migrations/default/1750352041274_fix_crash_match_trigger/up.sql

```diff
@@ -0,0 +1,166 @@
+CREATE OR REPLACE FUNCTION public.update_crash_ems_match()
+RETURNS trigger
+LANGUAGE plpgsql
+AS $function$
+DECLARE
+    matching_ems RECORD;
+    ems_record RECORD;
+    ems_cleanup_count INTEGER;
+    match_count INTEGER;
+    matched_crash_ids INTEGER[];
+    updated_matched_crash_ids INTEGER[];
+    remaining_crash_ids_count INTEGER;
+    meters_threshold INTEGER := 1200;
+    -- 30 min threshold equates to a 60 minute window (+/- 30 minutes of crash timestamp)
+    time_threshold INTERVAL := '30 minutes';
+BEGIN
+    -- Find all EMS records near the crash location + time
+    FOR matching_ems IN (
+        SELECT 
+            e.id,
+            e.incident_received_datetime,
+            e.geometry,
+            e.crash_match_status
+        FROM 
+            ems__incidents e
+        WHERE
+            e.incident_received_datetime  >= (NEW.crash_timestamp - time_threshold)
+            AND e.incident_received_datetime  <= (NEW.crash_timestamp + time_threshold)
+            AND e.geometry IS NOT NULL
+            AND NEW.position IS NOT NULL
+            AND ST_DWithin(e.geometry::geography, NEW.position::geography, meters_threshold)
+    ) LOOP
+        -- Find all crashes which match this EMS record location + time
+        SELECT 
+        COALESCE(array_agg(c.id), ARRAY[]::integer[])
+        FROM crashes c
```

Comment:
Small change here to use `coalesce(array_agg())` as discussed [here](https://github.com/cityofaustin/vision-zero/pull/1795/files#r2136399285)